src/custom/hooks/useMarkdown.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react'
+
+export default function useMarkdown(file: string) {
+  const [content, setContent] = useState('')
+
+  useEffect(() => {
+    const fetchContent = async () => {
+      await fetch(file)
+        .then(res => res.text())
```

Comment:
You don't need 2 thens, right?

Reply:
Actually it's better/cleaner I think, otherwise you have to make the current function `async` and `await` the `res.text()` from my testing.

Reply:
Because the return value is a Promise and not the actual string... it's only a string in the next upcoming `.then.`

Reply:
okok, fine good then